elastica/mesh/mesh_initializer.py

```diff
@@ -127,14 +127,16 @@ def face_calculation(
         Notes:
         ------
 
-        - This function has been tested for triangular meshes only.
+        - This function works only if each face of the mesh has equal no. of vertices i.e
+          all the faces of the mesh has similar geometry.
         """
-        faces = np.zeros((3, 3, n_faces))
+        no_of_vetices_in_each_face = pvfaces[0]
```

Comment:
typo

Reply:
```suggestion
        n_of_vertices_in_each_face = pvfaces[0]
```
Lets use `n` instead of `no` to be consistent